### DISCUSSION TITLE
Ansible Playbook Help

### DISCUSSION THREAD

**Discussion by spudgun1001** [hnimminh/libresbc#171] [Q&A] [answered]
Please see the below screenshot.

Am I missing something? I have configured the ansible.cfg file with my vault, ssh key and user.

What do I need to put in the vault? I am confused, there is not a lot of documentation regarding what exactly I need to put in the vault.

Please can I get some guidance? I swapped from Docker to this in hopes more people could assist.

![image](https://github.com/user-attachments/assets/cd42ec37-1323-4f53-b262-4627fbabb0f5)

**Comment by hnimminh** [comment]
Hi @spudgun1001 it clearly show that ssh is failed. Are you able to access to the host via ssh with the same credential (what you config in ansible)

**Reply by spudgun1001** [reply]
Where do I need to declare the login credentials in the config? I can SSH fine.

**Reply by hnimminh** [reply]
you can declare in `build/ansible/ansible.cfg` with below fields: 

- private_key_file 
- remote_port 
- remote_user

or specify per host in `build/ansible/inventories/production/hosts.yml`  with below fields: 
- ansible_user
- ansible_ssh_private_key_file
- ansible_port

**Reply by spudgun1001** [reply]
I have already declared my:
private_key_file
remote_port
remote_user

However I continue to get the error. I have my SSH key, I have declared the remote port and user.

**Reply by hnimminh** [reply]
it's just an ansible.
Could you please screenshot, how you config ansible and how you ssh to the server.  It likely you missed the configuration.

**Reply by spudgun1001** [reply, marked as answer]
I have found a fix.

By going to the hosts.yml file and declaring the following:

      ansible_user: 
      ansible_ssh_private_key_file: 
      ansible_port: 
      ansible_ssh_pass: 
      
 Has fixed my issue, Ansible can connect on.
